src/actions/profile-view.js

```diff
@@ -168,13 +168,15 @@ export function showGlobalTrack(trackIndex: TrackIndex): ThunkAction<void> {
   };
 }
 
-export function isolateGlobalTrack(
+/**
```

Comment:
I split `isolateGlobalTrack` into `isolateProcess` and `isolateProcessMainThread`.